src/main/webapp/paginationScript.js

```diff
@@ -56,7 +56,19 @@ function displayCards(listAugment) {
 
     resultsContent.innerHTML = '';
     for (let i = currentFirstCardIndex; i < (currentFirstCardIndex + TOTAL_CARDS_TO_DISPLAY); i++) {
+      if (photoReferencesArray[i] != 'none') {
+        loadImage(listingsArray[i].children[0].children[0], photoReferencesArray[i]);
```

Comment:
nit: a comment here explaining what the .children[0].children[0] is representing may be helpful

Reply:
Better yet, I added a variable imageElement, with a comment to explain further